updated README.md with neccessary steps for setup

seabeePraveen opened the pull request:
Issue No. : #27

abe-101 commented on README.md:
````diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
+  createdb --username=<USERNAME> my_hebrew_dates
+  ```
+
+6. Create Env Files and add these:
````
please change to 'Create `.env` Files and add these'

seabeePraveen replied:
yeah okay

abe-101 commented on README.md:
```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
+</a>
+</div>
+
 ## Settings
```
remove this section

seabeePraveen replied:
as u have mentioned for the credit of contributers

abe-101 commented on README.md:
```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
 
 Moved to [settings](http://cookiecutter-django.readthedocs.io/en/latest/settings.html).
 
 ## Basic Commands
```
remove this also

seabeePraveen replied:
u mean settings??

abe-101 replied:
yes

abe-101 commented on README.md:
```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
+
 ### Setting Up Your Users
 
 - To create a **normal user account**, just go to Sign Up and fill out the form. Once you submit it, you'll see a "Verify Your E-mail Address" page. Go to your console to see a simulated email verification message. Copy the link into your browser. Now the user's email should be verified and ready to go.
```
i think we can remove 
### Type checks 
and
### Test coverage

seabeePraveen replied:
sry could u specify the line number or point number

abe-101 replied:
we can remove lines 79 though 127 (new line numbers)

abe-101 replied:
@seabeePraveen Can you make the final fix's and I'll merge?

seabeePraveen replied:
yeah yeah i will make the final fix's
and sry to ask again like i found the setup was fine so could u please specify why these changes are required for those lines u have mentioned also please mention from which line to which line i have to change

seabeePraveen commented:
hii @abe-101 i have made all the changes as you have asked:
1. removed the settings section
2. changes the evn line
3. removed the lines from the type checks to the bottom

please let me know if i want to make any changes
Thank You!

seabeePraveen commented:
Happy Coding!

abe-101 commented:
@seabeePraveen Thank you very much!
